docs/SUMMARY.md

```diff
@@ -15,7 +15,6 @@
   * [Feature view](getting-started/concepts/feature-view.md)
   * [Feature retrieval](getting-started/concepts/feature-retrieval.md)
   * [Point-in-time joins](getting-started/concepts/point-in-time-joins.md)
-  * [Registry](getting-started/concepts/registry.md)
```

Comment:
I think it still makes sense to keep it here in the concepts though.

Reply:
bump on this. i think in another PR we can add metadata as the registry in the arch section

Reply:
yeah, agreed on that. Do you think we need to make any related changes in this PR as well?

Reply:
Let me clarify. I think we should still keep registry in the components section.

Reply:
So in this pr it should be in concepts and in another PR we can add metadata in architecture (which is the construct that registry aims to implement).

Reply:
Oh I see you moved it to components. We're good.